src/main/java/seedu/mark/model/reminder/Reminder.java

```diff
@@ -15,7 +15,7 @@
 public class Reminder {
 
     //TODO: temporarily; to change when confirmed
-    private static final String DATE_FORMATTER = "dd/MM/yyyy HHmm";
+    private static final String DATE_FORMATTER = "dd/MM/uuuu HHmm";
```

Comment:
Instead of have so many date formatters, can have all the other constants set to this one, so that don't have to repeat so many,

Reply:
I am thinking that the reminder only take the LocalDateTime object, so I leave the parsing part time string to parseUtil. (It also sound more reasonable)However, when generating the formatted time string, the reminder need to use the formatter again. I haven't decided where is the best place to put the formatter.

Reply:
yeah ok is just a small issue